frontend/src/components/ui/DifficultyButtonGroup.js

```diff
@@ -0,0 +1,21 @@
+import { ButtonGroup, Container } from "@mui/material";
+
+import React from "react";
```

Comment:
You may remove this line since `import React from "react";` is done automatically by react. Only need to import when you are using hook provided from "react" such as useState, useEffect, etc